src/components/BuildingsListPage/buildingslist.js

```diff
@@ -0,0 +1,653 @@
+      zoom: 13
+    })
+
+    const detailSection = document.querySelector('.detail-popup__overflow')
+    detailSection.innerHTML = ''
+    let html = `
+    <div class="detail-popup__row">
+        <div class="detail-popup__img-container">
+          <img class="detail-popup__img" src="${item[0].properties.image}" alt="" />
+          <img class="watch__icon" src="${watchIcon}" alt="icon watch image">
+        </div>
+
+        <div class="detail-popup__address">`
+
+    if (item[0].properties.name != null && item[0].properties.name !== 'null') {
+      html += `<h1 class="detail-popup__name">${item[0].properties.name}</h1>`
+    }
+
+    html += `
+            <h2 class="detail-popup__address__street"> ${item[0].properties.street} ${item[0].properties.number} </h2>
+            <h2 class="detail-popup__address__municipality">${item[0].properties.zip_code} ${item[0].properties.city}</h2>
+        </div>
+      </div>
+      <div class="detail-popup-info">`
+
+    if (item[0].properties.styles != null && item[0].properties.styles !== 'null') {
+      html += `
+        <div class="detail-popup__tags-group">
+          <div class="tag__category">
+           <img class="tag__icon" src="${styleIcon}" alt="icon style tags">
+           <h3 class="tag__group-name">Styles</h3>
+           <div class="line line--style"></div>
+          </div>
+           <div class="detail-popup__tags">
+               <div class="tag tag--style">${item[0].properties.styles}</div>
+           </div>
+       </div> `
+    }
+
+    if (item[0].properties.typographies != null && item[0].properties.typographies !== 'null') {
+      html += `
+      <div class="detail-popup__tags-group">
+            <div class="tag__category">
+              <img class="tag__icon" src="${typeIcon}" alt="icon type tags">
+              <h3 class="tag__group-name">Types</h3>
+              <div class="line line--type"></div>
+            </div>
+              <div class="detail-popup__tags">
+                  <div  class="tag tag--type">${item[0].properties.typographies}</div>
+            </div>
+        </div>`
+    }
+
+    if (item[0].properties.intervenants != null && item[0].properties.intervenants !== 'null') {
+      html += `
+        <div class="detail-popup__tags-group">
+          <div class="tag__category">
+            <img class="tag__icon" src="${architectIcon}" alt="icon architect tags">
+            <h3 class="tag__group-name">Architects</h3>
+            <div class="line line--architect"></div>
+          </div> 
+          <div class="detail-popup__tags">
+              <div class="tag-group-architect">
+                  <div class="tag tag--architect">${item[0].properties.intervenants}</div>
+              </div>
+          </div>
+        </div>`
+    }
+
+    html += `
+       </div>
+        <a href="${item[0].properties.url}" class="button button--dark" >Get to know more</a> 
+      </div>`
+
+    detailSection.innerHTML = html
+  }
+
+}
```

Comment:
Shouldn't this be in the building detail page component ?

Reply:
Okay, apparently building detail and building list are in the same component ! 👌